lua/core/menu/devices.lua

```diff
@@ -75,8 +76,8 @@ m.key = function(n,z)
         hid.update_devices()
       end
       m.mode = "list"
-      m.len = 4
-      m.pos = 1
+      m.len = m.section == "midi" and 16 or 4
```

Comment:
Doesn't this mean the same thing as #L38 and #L60? Just wondering why the logic was flipped here. When I was skimming this at first they looked like weird magic numbers but I get it now (e.g. grid vs. MIDI).

Reply:
good eye! just changed mid-code, no reason for the break. will unify :)